assets/js/googlesitekit/data/create-settings-store.test.js

```diff
@@ -131,29 +130,20 @@ describe( 'createSettingsStore store', () => {
 		describe( 'saveSettings', () => {
 			it( 'does not require any params', () => {
 				expect( async () => {
-					fetch
-						.doMockOnceIf(
-							/^\/google-site-kit\/v1\/core\/site\/data\/settings/
-						)
-						.mockResponseOnce(
-							JSON.stringify( { setting1: 'serverside' } ),
-							{ status: 200 }
-						);
-
+					fetchMock.getOnce(
+						/^\/google-site-kit\/v1\/core\/site\/data\/settings/,
+						{ body: { setting1: 'serverside' }, status: 200 }
+					);
```

Comment:
Oh yeah that makes sense, there's an unmatched GET with that as a `postOnce` though. I think that's being triggered by the the `getSettings()` call inside `saveSettings()`, so I added a `receiveSettings()` before to avoid making that extra GET request.

Reply:
Actually, on second thought, I think it makes more sense to just mock theGET request also.